### DISCUSSION TITLE
Baresip with voip.ms

### DISCUSSION THREAD

**Discussion by wgmayer0** [baresip/baresip#2448] [General]
Has anyone gotten Baresip working with voip.ms? I have a voip ms account and when I try to dial Baresip says: "could not find UA for [phone number]"

**Comment by wgmayer0** [comment]
Figured it out. I had [user number]@voip.ms when it should be [user number]@chicago1.voip.ms

**Comment by bjlockie** [comment]
I need help setting it up.
Is the sip account  [user number]@chicago1.voip.ms and the Telephony Provider is chicago1.voip.ms for you?
I tried mine [sub account and account]@toronto9,voip.ms for the sip account and the telephony provider and it fails to register.

**Reply by wgmayer0** [reply]
The exact line I used, replacing user data with `X`s, was: ```<sip:XXXXXX@chicago4.voip.ms;transport=tcp>;auth_pass=XXXXXXXX```

**Reply by wgmayer0** [reply]
The file is located at ```./.baresip/accounts```

**Reply by bjlockie** [reply]
Great, thanks.

**Comment by bjlockie** [comment]
I found an Android project with a similar name.
That is what I want. :-)
